docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# The first instance will automatically bootstrap the cluster.
+# Scale this tier and each additional instance will automatically
+# self-configure as a member of the cluster
+couchbase:
+    image: autopilotpattern/couchbase:latest # enterprise-4.0.0-r1
```

Comment:
yes, I was using `<their version number><our revision of that version>` `r3` makes sense